FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+* [Fixed] Potentially drop pre-warmed app start traces on iOS 15 and above (#9026). App start measurements are made only for cold app starts (without pre-warming).
```

Comment:
Nit: Could you change it to lower case `fix`? Not sure if this will have any impact. Just to make sure we align with all Firebase CHANGELOG.md.

Reply:
Done